RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -648,8 +648,8 @@ boolean doBedtime()
 	januaryToteAcquire($item[Makeshift Garbage Shirt]);		//doubles stat gains in the LOV tunnel. also keep leftover charges for tomorrow.
 	loveTunnelAcquire(true, $stat[none], true, 3, true, 1);
 
-	if(item_amount($item[Genie Bottle]) > 0)
-	{
+	if(item_amount($item[Genie Bottle]) > 0 && auto_is_valid($item[genie bottle]) && auto_is_valid($item[pocket wish]) && !in_glover())
+	{ // can't use genie bottle in BHY and can't use pocket wishes in G-Lover
```

Comment:
the auto_is_valid does not need a comment saying it is invalid in BHY.
just because pocket wishes do not work in glover does not mean we should not make them in bedtime.